src/modules/DisplayModule.cpp

```diff
@@ -0,0 +1,73 @@
+#include "DisplayModule.h"
+#include "../Renderer.h"
+#include <cstdio>
+#include <functional>
+
+DisplayModule::DisplayModule(ModularSynth& synth)
+	:SynthModule(synth, moduleId, 1, 1, 0), floatInput(0)
+{
+}
+
+
+void DisplayModule::cycle()
+{
+	floatInput = getInput(0);
+	
+	// Passthru
+	setOutput(0, getInput(0));
+}
+
+
+
+const char * DisplayModule::getInputName(int input) const
+{
+	static const char *names[] = {"Input"};
+	return names[input];
+}
+
+
+const char * DisplayModule::getOutputName(int output) const
+{
+	static const char *names[] = {"Passthru"};
+	return names[output];
+}
+
+
+const char * DisplayModule::getName() const
+{
+	return "Display";
+}
+
+void DisplayModule::onLoaded()
+{
+	setInput(0, 0.0f);
+	stringMeUp(getInput(0));
+}
+
+
+char * DisplayModule::stringMeUp(float value) const
+{
+	static char funky[20];
```

Comment:
I suggest using a better variable name, it will save you in the future. ;)

Even just something like "temp" is better because it clearly says it's something temporary.